src/components/ProfileCard/index.js

```diff
@@ -4,11 +4,11 @@ import Text from "../Text";
 import { LogoTwitter, LogoLinkedin, LogoGithub } from "@carbon/icons-react";
 
 const ProfileCard = ({ photoUrl, name, role, description, twitter, linkedin, github, reverse }) => {
+  let isDescription = description === undefined ;
```

Comment:
1) You can use `const` here.
2) Variable name is wrong, `description === undefined` when **there is no descrtiption**. You should either name variable `noDescription` or check `description !== undefined`. If you choose the other option, you will also need changes further in code.